plutus-use-cases/test/Spec/Governance.hs

```diff
@@ -69,7 +69,7 @@ params = Gov.Params
     , Gov.baseTokenName = baseName
     }
 
-lawv1, lawv2, lawv3 :: ByteString
+lawv1, lawv2, lawv3 :: BuiltinByteString
```

Comment:
I think you can just revert all the changes in this file now

Reply:
Unfortunately, it will require even more changes because `lawv1`, `lawv2` and `lawv3` are used above in `checkPredicate` and they expect `BuiltinByteString` there. And in `votingRound` `law` has to be `BuiltinByteString`.

So I choose the way with the minimal amount of diffs...

Reply:
Ah okay, I thought this was solved by your change to the endpoint to expect a normal bytestring.